projects/packages/videopress/src/client/admin/components/video-filter/index.tsx

```diff
@@ -112,16 +124,19 @@ export const FilterSection = ( props: {
 						for="filter-g"
 						label={ __( 'G', 'jetpack-videopress-pkg' ) }
 						onChange={ checked => props.onChange?.( 'rating', 'G', checked ) }
+						checked={ filterIsChecked( 'rating', 'G' ) }
```

Comment:
We should probably use constants here.
https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/state/constants.js#L86